tools/app-provisioning-tool/app-provisioning-tool/ms-identity-app.csproj

```diff
@@ -5,12 +5,39 @@
     <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
     <Nullable>enable</Nullable>
     <RootNamespace>Microsoft.Identity.App</RootNamespace>
-    
+
     <PackAsTool>true</PackAsTool>
     <ToolCommandName>ms-identity-app</ToolCommandName>
     <PackageOutputPath>./nupkg</PackageOutputPath>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../../../build/MSAL.snk</AssemblyOriginatorKeyFile>
+
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <!-- Copyright needs to be in the form of © not (c) to be compliant -->
+    <Title>ms-identity-app</Title>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft Corporation</Company>
+    <Product>ms-identity-app</Product>
+    <Description>
+      This package enables ASP.NET Core web apps and web APIs to use the Microsoft identity platform (formerly Azure AD v2.0).
```

Comment:
This package is a dotnet global tool which registers Azure AD or Azure AD B2C applications, and updates your code accordingly. For details see https://aka.ms/ms-identity-app-registration

Reply:
right, had not even looked at this. thanks for the link.

Reply:
made a few more changes here